certora/specs/RevertsERC20.spec

```diff
@@ -0,0 +1,47 @@
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+methods {
+    function balanceOf(address) external returns uint256 envfree;
+    function allowance(address, address) external returns uint256 envfree;
+
+    // Avoids checking delegation related overflows.
+    function _._moveDelegateVotes(address, address, uint256) internal => CONSTANT;
```

Comment:
This would be a good time to make sure that this cannot happen, since this PR is about reverts

Reply:
Good idea, this is a nice intermediate step :fire:

Reply:
Implemented in https://github.com/morpho-org/morpho-token-upgradeable/pull/98/commits/a61550f74f2d08419d12aaf5204aaebb8a085608.